api/v1/flow/flow.proto

```diff
@@ -375,6 +375,10 @@ message FlowFilter {
     repeated string http_method = 21;
     // http_path is a list of regular expressions to filter on the HTTP path.
     repeated string http_path = 22;
+
+    // node_names is a list of patterns to filter on the node name, e.g. (e.g.
+    // "k8s*", "test-cluster/*.domain.com", "cluster-name/**").
+    repeated string node_names = 23;
```

Comment:
Every other field name in `FlowFilter` is not pluralized although they're all repeated fields. Please, change to `node_name` for consistency.

Reply:
Done.